.travis.yml

```diff
@@ -67,6 +67,11 @@ jobs:
           script: FIREFOX_WD_URL=http://localhost:4444/wd/hub CHROME_WD_URL=http://localhost:4445/wd/hub npm run lerna:publish
           on:
             tags: true
+        - provider: script
+          skip_cleanup: false
+          script: lerna run typedoc
+          on:
+            tags: true
```

Comment:
Are you sure that all dev dependencies are still available at this point?
Due to `skip_cleanup: false` in the previous steps and the step itself, all changes to the repo should be `git stash --all` including the `node_modules` folder containing the typedoc dependency. As `lerna` is installed globally, this is not an issue.

Maybe we should change `skip_cleanup` to `true`.

Reply:
The [docs on travis](https://docs.travis-ci.com/user/deployment/pages/) state:
> Make sure you have `skip_cleanup` set to `true`